__tests__/app.test.js

```diff
@@ -126,3 +126,48 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("200: Responds with the array with the most recent comments first",()=>{
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body:{comments}})=>{
+      expect(comments).toBeSortedBy('created_at',{descending:true})
+    })
+  })
+  test("404: Article_id does not exist",()=>{
+    return request(app)
+    .get("/api/articles/9001/comments")
+    .expect(404)
+    .then(({body:{message}})=>{
+      expect(message).toBe("Not Found")
+    })
+  })
+  test("400: Article_id is not a number",()=>{
+    return request(app)
+    .get("/api/articles/its-over-nine-thousand/comments")
+    .expect(400)
+    .then(({body:{message}})=>{
+      expect(message).toBe("Bad request")
+    })
```

Comment:
What about an article with no comments, what status code should that be? I would argue it isn't an error for an array to be empty if the article id is valid.